.github/workflows/rpm.yml

```diff
@@ -153,6 +153,10 @@ jobs:
           name: source0
           path: /tmp/rpmbuild/SOURCES/
 
+      - name: Generate about page information
```

Comment:
This will need moved up to the source0 stage so that it gets included in the source0 tarball.